<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="visitno_spin">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="visitno_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>VisitNo</string>
   </property>
  </widget>
  <widget class="QComboBox" name="vtype_box">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="type_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="vtimestamp_edit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>194</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="who_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>WHO</string>
   </property>
  </widget>
  <widget class="QComboBox" name="study_box">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="study_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Study</string>
   </property>
  </widget>
  <widget class="QLabel" name="cohort_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Cohort</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cohort_edit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Control</string>
   </property>
  </widget>
  <widget class="QLabel" name="dur_hr_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration (hours)</string>
   </property>
  </widget>
  <widget class="QLabel" name="note_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Note</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="note_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>461</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dur_hr_spin">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
